.gitlab-ci.yml

```
@@ -100,6 +100,23 @@ test:plugin:
     reports:
       junit: dd-sdk-android-gradle-plugin/build/test-results/test/TEST-*.xml
 
+test:instrumented:
+  tags: [ "arch:amd64" ]
+  image: $CI_IMAGE_DOCKER
+  stage: test
+  timeout: 1h
+  script:
+    - git fetch --depth=1 origin main
+    - rm -rf ~/.gradle/daemon/
+    - CODECOV_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android-gradle-plugin.codecov-token --with-decryption --query "Parameter.Value" --out text)
+    - export DD_AGENT_HOST="$BUILDENV_HOST_IP"
+    - GRADLE_OPTS="-Xmx2560m" DD_TAGS="test.module:instrumented" ./gradlew :instrumented:test --stacktrace --no-daemon -Dorg.gradle.jvmargs=-javaagent:$(pwd)/libs/dd-java-agent-1.26.1.jar=$DD_COMMON_AGENT_CONFIG
+    - bash <(cat ./codecov.sh) -t $CODECOV_TOKEN
+  artifacts:
+    reports:
+      junit: dd-sdk-android-gradle-plugin/build/test-results/test/TEST-*.xml
```

path here will be different

```suggestion
      junit: instrumented/build/test-results/test/TEST-*.xml
```

---

updated